pkg/agent/route/route_linux.go

```diff
@@ -110,24 +126,34 @@ func (c *Client) Initialize(nodeConfig *config.NodeConfig, done func()) error {
 	return nil
 }
 
-// initIPTablesOnce starts a loop that initializes the iptables infrastructure.
-// It returns after one successful execution.
-func (c *Client) initIPTablesOnce(done func()) {
-	defer done()
-	backoffTime := 2 * time.Second
+// Run waits for iptables initialization, then periodically syncs iptable rules.
+// It will not return until stopCh is closed.
+func (c *Client) Run(stopCh <-chan struct{}, syncInterval time.Duration) {
 	for {
```

Comment:
is that a busy loop? why don't we use a channel instead?

Reply:
Ya we should 👍🏼 , updated